prerequisites/software.rst

```diff
@@ -162,6 +162,12 @@ You can choose between the following database servers:
 
    .. include:: /appendix/includes/mysql-deprication-note.rst
 
+.. note::
+   Make sure to use session connection pooling. Transaction-based connection
+   pooling (e.g. by using PgBouncer) is not supported and may lead to errors
+   during database migrations.
+
+
```

Comment:
Wording suggestion to make it more clear what it refers to:

If you use database connection pooling software like PgBouncer, make sure to use a pooling mode that is fully compatible with PostgreSQL. Typically this is called "session connection pooling". Transaction-based connection 
 pooling is not supported and may lead to errors during database migrations.